commerce_sendcloud.module

```diff
@@ -399,7 +403,6 @@ function commerce_sendcloud_create_label($order) {
           if ($docall) {
             // Approach the api.
             $sendcloud_api = new SendcloudApi(
-              'live',
```

Comment:
According to the constructor of the PHP wrapper, not anymore :) 

(Weird construction BTW, but I can understand the need to support legacy code .

![selection_999 688](https://cloud.githubusercontent.com/assets/5662679/15137608/3644257c-168b-11e6-85cb-9365c0e86e80.png)